eng/resolveContract.targets

```diff
@@ -62,6 +63,22 @@
     <_ApiCompatLibReplacementString Condition="'$(MSBuildProjectName)' == 'System.Private.CoreLib'">lib/$(NetCoreAppCurrent)/$2</_ApiCompatLibReplacementString>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <IsRuntimeAndReferenceAssembly Condition="'$(IsRuntimeAndReferenceAssembly)' == '' and
+                                              '$(HasMatchingContract)' != 'true' and
+                                              '$(IsPrivateAssembly)' != 'true'">true</IsRuntimeAndReferenceAssembly>
```

Comment:
Can you give me an example of a private assembly?

Reply:
System.Private.Uri